client/app/bundles/server-components/components/ServerInfo.jsx

```
@@ -0,0 +1,58 @@
+  };
+
+  return (
+    <div className="bg-gradient-to-br from-emerald-50 to-teal-50 border border-emerald-200 rounded-xl p-6">
+      <p className="text-xs text-emerald-600 mb-4 font-medium">
+        This data comes from the Node.js <code className="bg-emerald-100 px-1 rounded">os</code> module
+        — it runs only on the server. The <code className="bg-emerald-100 px-1 rounded">lodash</code> library
+        used to format it never reaches the browser.
+      </p>
+      <div className="grid md:grid-cols-2 gap-x-8 gap-y-1">
+        {grouped.map((group, gi) => (
+          <div key={gi} className="space-y-1">
```

ESLint

[error] Do not use Array index in keys

(react/no-array-index-key)